test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/project.json

```diff
@@ -0,0 +1,33 @@
+﻿{
+  "buildOptions": {
+    "warningsAsErrors": true
+  },
+
+  "dependencies": {
+    "dotnet-test-xunit": "2.2.0-*",
+    "Microsoft.AspNetCore.Http": "1.1.0",
+    "Microsoft.AspNetCore.Sockets": "0.1.0-*",
+    "Microsoft.AspNetCore.Hosting": "1.1.0-*",
+    "Microsoft.AspNetCore.Server.Kestrel": "1.1.0-*",
+    "Microsoft.AspNetCore.SignalR": {
+      "target": "project"
+    },
+    "Microsoft.AspNetCore.SignalR.Client": {
+      "target": "project"
```

Comment:
Yeah, even though it sounds like the right thing to do, Universe removes the source projects in order to force the tests to run against packages, so this constraint will cause a problem.